user-guide/Troubleshooting/Known_issues/KI_Problem_after_OS_upgrade_to_win_2022.md

```diff
@@ -21,21 +21,8 @@ After an upgrade to Windows Server 2022, the following issues can cause problems
 
 - For the IIS issue:
 
-  1. Enable IIS. See [How to Enable IIS on Windows Server](https://techcommunity.microsoft.com/t5/iis-support-blog/how-to-enable-iis-and-key-features-on-windows-server-a-step-by/ba-p/4229883).
-
-  1. Configure the default website:
-
-     1. Open *IIS Manager*.
-
-     1. In the *Connections* pane on the left, expand the top node and *Sites* node until you see *Default Web Site*.
-
-     1. Under *Actions* on the right, click *Basic Settings*.
-
-     1. Make sure the settings are filled in as follows, and click *OK*.
-
-        ![Default website configuration](~/user-guide/images/KI_IIS_default_website.png)
-
-  1. Restart IIS.
+  1. On the DMA server, open Server Manager, and make sure that IIS is enabled and that, in addition to the default selections, the *WebSockets* and *IIS 6 Management Compatibility* features are selected.
```

Comment:
Looks like the default selections depend on the version of Windows Server, might be safer to use the following PowerShell command: `Add-WindowsFeature Web-Server, Web-WebServer, Web-Security, Web-Filtering, Web-ASP, Web-Asp-Net45, Web-Mgmt-Compat, Web-Lgcy-Scripting, Web-ISAPI-Filter, Web-WMI, Web-WebSockets`

Reply:
Is this what you had in mind?
```suggestion
  1. On the DMA server, open Server Manager, and make sure that IIS is enabled.
  1. Use the following PowerShell command to configure the necessary IIS features:
  
     ```txt
     Add-WindowsFeature Web-Server, Web-WebServer, Web-Security, Web-Filtering, Web-ASP, Web-Asp-Net45, Web-Mgmt-Compat, Web-Lgcy-Scripting, Web-ISAPI-Filter, Web-WMI, Web-WebSockets
     ```
```

Reply:
I am wondering if it isn't better to appoint Wim as reviewer and remove me?

Reply:
Yes indeed, that would be better